web/themes/custom/pineapple/sass/layouts/nodes/_author.scss

```diff
@@ -1,52 +1,111 @@
-.author-container {
-  margin-bottom: $unit *4;
-  padding: $unit $unit*3;
-  background-color: $light-gray;
+.author-body {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
 
   @include breakpoint($lap) {
-    display: flex;
     flex-direction: row;
+    align-items: flex-start;
+    margin-top: $unit *4;
   }
 
-  .author-info-container {
+  .sidebar-container {
```

Comment:
On mobile there should be more space between the side bar and my content heading

Reply:
done